### ISSUE TITLE
What's the accuracy of CvT-13 without pre-trained on CIFAR10 

### ISSUE THREAD

**Issue by hanwenran1** [microsoft/CvT#25]
Hi,

What's the accuracy of CvT-13 without pre-trained on CIFAR10? Mine is only 79.6. Would you like to told me yours? And what are the hyper-parameters for fine-tuning on CIFAR10 without pre-trained ? I can't find it in detail in the paper.

Thanks.
